desk/app/groups.hoon

```diff
@@ -1240,6 +1240,12 @@
   ::
       [%server %groups ship=@ name=@ rest=*]
     =+  ship=(slav %p ship.pole)
+    ::  ignore responses after group has been deleted 
+    ::
+    ?:  ?&  !(~(has by groups) ship name.pole)
+            ?=([%invite %revoke ship=@ ~] rest.pole)
+        ==
+      cor
```

Comment:
And we ignore only these because getting signs on other wires is unexpected when the group is no longer in state, right?